### DISCUSSION TITLE
Responding to REST?

### DISCUSSION THREAD

**Discussion by meeech** [toverainc/willow#167] [Q&A] [answered]
Hi. Just wondering if there's any beta functionality to respond to the REST request where it delivers the text? For example, to have it display something on screen? Or play some delivered audio?

**Comment by kristiankielhofner** [comment, marked as answer]
With REST endpoint support we will display the REST endpoint HTTP response as-is on the display. Additionally, if you are configured for playing tone confirmation it will play either success or failure tone based on the HTTP response status code (HTTP 2xx for success, anything else is error).

If you are configured for WIS text to speech output it will also speak the text returned from your REST endpoint. It works just like the HA or openHAB endpoints.

**Reply by meeech** [reply]
Cool! Will give that a shot then
